### PR TITLE
Reword hybrid taat/daat documentation

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: en/performance/feature-tuning.html [hunk 1]
```diff
@@ -77,25 +77,78 @@ <h2 id="when-to-use-fast-search">When to use fast-search for attribute fields</h
...
 </p>
 
-<h2 id="hybrid-taat-daat">Hybrid TAAT (term-at-a-time) and DAAT (document-at-a-time) query evaluation</h2>
+<h2 id="hybrid-taat-daat">Hybrid TAAT and DAAT query evaluation</h2>
 <p>
-Generally Vespa does <em>DAAT</em> (document-at-a-time) query evaluation and not <em>TAAT</em> (term-at-a time) so ranking and matching is not 
-fully two separate phases where one first find matches and later ranks them.
-Matching and first-phase score calculation is interleaved when using DAAT. 
-</p>
+Vespa support <b>hybrid</b> query evaluation over inverted indexes combining <em>TAAT</em> and <em>DAAT</em> evaluation to combine the best of both
```
Comment (kkraune) [inline en/performance/feature-tuning.html:82]:
```suggestion
Vespa supports <strong>hybrid</strong> query evaluation over inverted indexes combining <em>TAAT</em> and <em>DAAT</em> evaluation to combine the best of both 
```

Comment (kkraune) [reply en/performance/feature-tuning.html:82]: minor: &lt;b&gt; is not supported in html5, I think

File: en/performance/feature-tuning.html [hunk 2]
```diff
@@ -77,25 +77,78 @@ <h2 id="when-to-use-fast-search">When to use fast-search for attribute fields</h
...
+Using <em>TAAT</em> can speed up query matching significantly (up to 20-30%) in cases where the query tree is large and complex 
+and where only parts of the query tree is used for term based ranking. Examples of branches that would require <em>DAAT</em> is using text ranking features 
+like <a href="../reference/rank-features.html">bm25/nativeRank</a>. The list of ranking features which can handle <em>TAAT</em> is long but  
+using <a href="../tensor-user-guide.html">attribute or tensor</a> features only can have the entire tree evaluated using <em>TAAT</em>.
+</p>
 
-Using TAAT can speed up query matching in some cases where the query tree is large and complex 
-and where only small parts of the query tree is used for ranking hits or ranking by a ranking expression only using <a href="../tensor-user-guide.html">attribute or tensor</a> features.
+<p>
+For example for a query where there is a user text query from an end user by one can use <em>userQuery()</em> YQL syntax
```
Comment (kkraune) [inline en/performance/feature-tuning.html:117]: Note sure if I understood the sentence - is this better?
```suggestion
For example, for a query where there is a user text query from an end user, one can use <em>userQuery()</em> YQL syntax 
```

**Review by kkraune — APPROVED** [review]
Great writeup!

**Comment by jobergum** [comment]
Thanks a lot for reading through @kkraune

**Pull request merged**